src/tests/functional/plugin/shared/src/behavior/plugin/life_time.cpp

```diff
@@ -79,12 +76,12 @@ namespace BehaviorTestsDefinitions {
 
     TEST_P(HoldersTest, Orders) {
         // Test failed if crash happens
-        EXPECT_NO_CRASH(release_order_test(order, targetDevice, function));
+        ASSERT_NO_THROW(release_order_test(order, targetDevice, function));
```

Comment:
@sbalandi @iefode I'm not sure that such approach might work here in the case of SegFault or smth like this.

Reply:
Hi, 
yes, it is not work for linux, but work for win